flink-clients/src/main/java/org/apache/flink/client/program/ContextEnvironment.java

```diff
@@ -84,7 +84,7 @@ public JobExecutionResult execute(String jobName) throws Exception {
 				client.getFlinkConfiguration(),
 				getParallelism());
 
-		ClientUtils.addJarFiles(jobGraph, this.jarFilesToAttach);
+		org.apache.flink.runtime.client.ClientUtils.addJarFiles(jobGraph, this.jarFilesToAttach);
```

Comment:
I think it's problematic that we have two `ClientUtils`, maybe this should be refactored/renamed before this change.

Reply:
Reverted.

BTW, in my opinion `submitJob`/`submitJobAndWaitForResult`/`executeProgram` are temporary batched utilities which can be reduced once Executor refactor ready.